File: src/server/gkill/api/req_res/update_timeis_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type UpdateTimeisResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedTimeis *reps.TimeIs `json:"updated_timeis"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`
}

File: src/client/classes/api/req_res/update-timeis-response.ts
'use strict'

import { TimeIs } from '@/classes/datas/time-is'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateTimeisResponse extends GkillAPIResponse {

    updated_timeis: TimeIs

    updated_kyou: Kyou | null

    constructor() {
        super()
        this.updated_timeis = new TimeIs()
        this.updated_kyou = null
    }

}


